src/ralph/virtual/models.py

```diff
@@ -267,6 +268,18 @@ def ip_addresses(self, value):
             base_object=self, ipaddress__address__in=to_delete
         ).delete()
 
+    @classmethod
+    @transition_action(
+        verbose_name=_('Cleanup security scans'),
+    )
+    def cleanup_security_scans(cls, instances, **kwargs):
```

Comment:
the same method is defined already for `DataCenterAsset`. Please extract it to some common class or put somewhere as plain function - `transition_action` has additional argument - `models` - see https://github.com/allegro/ralph/blob/ng/src/ralph/deployment/deployment.py for examples.